dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ClientRegressionIT.java

```diff
@@ -43,68 +40,70 @@
 import org.junit.experimental.categories.Category;
 import org.junit.rules.TemporaryFolder;
 
+import static io.dockstore.common.CommonTestUtilities.checkToolList;
 import static io.dockstore.common.CommonTestUtilities.runOldDockstoreClient;
+import static io.dockstore.common.CommonTestUtilities.version;
 
 /**
- * @author dyuen
+ *
  */
 @Category({ RegressionTest.class })
 public class ClientRegressionIT extends BaseIT {
-    static URL url;
-    static File dockstore;
-    static File testJson;
-
+    @ClassRule
+    public static TemporaryFolder temporaryFolder = new TemporaryFolder();
     @Rule
     public final SystemOutRule systemOutRule = new SystemOutRule().enableLog();
-
     @Rule
     public final SystemErrRule systemErrRule = new SystemErrRule().enableLog();
-
     @Rule
     public final ExpectedSystemExit systemExit = ExpectedSystemExit.none();
-
-    @ClassRule
-    public static TemporaryFolder temporaryFolder = new TemporaryFolder();
-
-    @Before
-    @Override
-    public void resetDBBetweenTests() throws Exception {
-        CommonTestUtilities.cleanStatePrivate1(SUPPORT);
-        Client.DEBUG.set(false);
-    }
+    static URL url;
+    static File dockstore;
+    static File testJson;
 
     @BeforeClass
     public static void getOldDockstoreClient() throws IOException {
-        url = new URL("https://github.com/ga4gh/dockstore/releases/download/" + CommonTestUtilities.version +"/dockstore");
+        url = new URL("https://github.com/ga4gh/dockstore/releases/download/" + CommonTestUtilities.version + "/dockstore");
         dockstore = temporaryFolder.newFile("dockstore");
         FileUtils.copyURLToFile(url, dockstore);
         dockstore.setExecutable(true);
         String[] commandArray = new String[] { "--version" };
         url = new URL("https://raw.githubusercontent.com/DockstoreTestUser/dockstore_parameter_test/master/test.cwl.json");
         testJson = temporaryFolder.newFile("test.cwl.json");
         FileUtils.copyURLToFile(url, testJson);
-//        This has problem executing for some reason
-//        ImmutablePair<String, String> stringStringImmutablePair = runOldDockstoreClient(commandArray);
+        // The below sometimes has a problem executing because of GitHub rate limit.  Disabling till fixed
```

Comment:
Oh, that's why. 
I think we can try to limit the refresh testing for regression to a minimal number of calls.